app/src/main/res/layout/activity_key_gear_item_detail.xml

```diff
@@ -40,15 +41,77 @@
                 android:layout_height="wrap_content"
                 android:layout_marginStart="@dimen/base_margin_triple"
                 android:layout_marginTop="@dimen/base_margin_quintuple"
-                android:paddingBottom="1500dp"
                 android:text="@string/KEY_GEAR_ITEM_VIEW_COVERAGE_TABLE_TITLE" />
 
+            <View
+                android:layout_width="match_parent"
+                android:layout_height="1dp"
```

Comment:
don't we have an dimen for this?

Reply:
We don't! We have 1dp all over the place it turns out. But I added one, lets try to use that one